### PR TITLE
one million engagement banner - content and partial styling

### PR DESCRIPTION
**Pull request by johnduffell**
## What does this change?
This adds the rough styling , plus switching between GB/ROW and existing or potential supporter.

This is still switced off but it seems sensible to ship the rough version so people can refine separately without having a big PR at the end.  It also gets an early review on the techniques used.

to see this use url hash on any page:
`/uk#noads&fiv`
to get the variants without messing with developer tools you can further append any of
```
&fiv-potential OR &fiv-existing
&fiv-uk OR &fiv-row
```
## Screenshots
So far I have only been working on the wide version
design is bottom, actual so far is above
image

---
same again for the potential supporters one
image

### Tested

- [ ] Locally
- [ ] On CODE (optional)

@tomrf1 @joelochlann

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js [hunk 1]
```diff
@@ -20,22 +30,52 @@ export const acquisitionsBannerFivTemplate = (
...
+            </div><div class="fiv-banner__headline fiv-banner__mobile">
+                ${potentialTitleMobile.markup}
+            </div>
+            <div class="fiv-banner__text_bold">
+                …we remain editorially independent, our journalism free from commercial bias and our reporting open and accessible to all.
+            </div>
+            <div class="fiv-banner__text_normal">
+                Imagine what we could continue to achieve with the support of many more of you. Together we can be a force for change.
+            </div>`
+                : `<div class="fiv-banner__headline">
```
Comment (tomrf1) [inline static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js]: this is hard to read with current formatting

File: static/src/stylesheets/module/site-messages/_fiv-banner.scss [hunk 2]
```diff
@@ -61,9 +62,23 @@
         line-height: 22px;
     }
 
-    @include mq(desktop) {
-        display: inline-block;
-    }
+}
+
+.fiv-banner__headline {
+
```
Comment (tomrf1) [inline static/src/stylesheets/module/site-messages/_fiv-banner.scss]: deliberately empty?

File: static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js (hunk 1 above)
Comment (joelochlann) [reply static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js]: `make fix` is not helpful in this case...

**Comment by joelochlann** [comment]
👍 let's merge and iterate on it!

File: static/src/stylesheets/module/site-messages/_fiv-banner.scss (hunk 2 above)
Comment (johnduffell) [reply static/src/stylesheets/module/site-messages/_fiv-banner.scss]: yes nothing there yet, todo

File: static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js [hunk 3]
```diff
@@ -20,22 +30,52 @@ export const acquisitionsBannerFivTemplate = (
...
     </div>
+    <div class="fiv-banner__cta">
+        <button class="button fiv-banner__button fiv-banner__button_contribute" href="${linkUrl}">
+            Contribute${arrowRight.markup}
+        </button>
+        ${
+            window.location.hash.match(/[#&]fiv-gb(&.*)?$/) ||
+            (geolocationGetSync() === 'GB' &&
+                !window.location.hash.match(/[#&]fiv-row(&.*)?$/))
+                ? `<button class="button fiv-banner__button fiv-banner__button_subscribe" href="${linkUrl}">
```
Comment (joelochlann) [inline static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js]: this has to be `<a>` element, not a `<button>`

File: static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js [hunk 4]
```diff
@@ -20,22 +30,52 @@ export const acquisitionsBannerFivTemplate = (
...
-                alt="PayPal and credit card"
-            >
+        <div class="fiv-banner__onwards">
+            <a class="fiv-banner__onwards" href="${linkUrl}">
+                Read our story${arrowRight.markup}
+            </a>
         </div>
     </div>
+    <div class="fiv-banner__cta">
+        <button class="button fiv-banner__button fiv-banner__button_contribute" href="${linkUrl}">
```
Comment (joelochlann) [inline static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js]: should be `<a>`

**Comment by johnduffell** [comment]
to see this use url hash
`http://localhost:9000/uk/media#noads&fiv`
you can further append any of
```
&fiv-potential OR &fiv-existing
&fiv-uk OR &fiv-row
```

File: static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js (hunk 3 above)
Comment (johnduffell) [reply static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js]: OK done this and removed the block link that overlays everything.  We still need the Read our story link though.

File: static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js [hunk 5]
```diff
@@ -20,25 +31,50 @@ export const acquisitionsBannerFivTemplate = (
...
-            <img
-                class="fiv-banner__payment-logos"
-                src="${config.get(
-                    'images.acquisitions.paypal-and-credit-card',
-                    ''
-                )}"
-                alt="PayPal and credit card"
-            >
+        <div class="fiv-banner__onwards">
+            <a class="fiv-banner__onwards" href="${linkUrl}">
```
Comment (johnduffell) [inline static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js]: this url is a TODO

**Pull request merged**